### PR TITLE
[zh] Sync install ambient with helm doc into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync install ambient with helm doc into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/ops/ambient/install/helm-installation/index.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+1. [安装 Helm 客户端](https://helm.sh/docs/intro/install/)，版本要求 3.6 或更高。
+
+1. 配置 Helm 仓库：
+
+    {{< text syntax=bash snip_id=configure_helm >}}
+    $ helm repo add istio https://istio-release.storage.googleapis.com/charts
+    $ helm repo update
+    {{< /text >}}
+
+有关命令文档，请参阅 [Helm 仓库](https://helm.sh/docs/helm/helm_repo/)。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/install/helm-installation/index.md:28]:
```suggestion
**有关 Helm 命令文档，请参阅 [Helm 仓库](https://helm.sh/docs/helm/helm_repo/)。**
```

File: content/zh/docs/ops/ambient/install/helm-installation/index.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+## 安装组件 {#installing-the-components}
+
+### 安装基础组件 {#installing-the-base-component}
+
+`base` Chart 包含设置 Istio 所需的基本 CRD 和集群角色。
+需要在任何其他 Istio 组件之前被安装。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/install/helm-installation/index.md:35]:
```suggestion
需要先安装此 Chart，才能安装任何其他 Istio 组件。
```

File: content/zh/docs/ops/ambient/install/helm-installation/index.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+1. 列出安装在 `istio-system` 命名空间中的所有 Istio Chart：
+
+    {{< text syntax=bash >}}
+    $ helm ls -n istio-system
+    NAME       NAMESPACE    REVISION UPDATED         STATUS   CHART        APP VERSION
+    istio-base istio-system 1        ... ... ... ... deployed base-1.0.0   1.0.0
+    istiod     istio-system 1        ... ... ... ... deployed istiod-1.0.0 1.0.0
+    {{< /text >}}
+
+1.（可选）删除所有 Istio 网关 Chart 安装：
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/install/helm-installation/index.md:111]:
```suggestion
1.（可选）删除所有 Istio 网关 Chart 安装文件：
```

**Review by windsonsea — COMMENTED** [review]
some text suggestions

File: content/zh/docs/ops/ambient/install/helm-installation/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/docs/ops/ambient/install/helm-installation/index.md:28]: Seems the whole Chinese sentence within the ** is not allowed by some lint rule, let's try.

Comment (windsonsea) [reply content/zh/docs/ops/ambient/install/helm-installation/index.md]: Well, understand. Your choice is good.

File: content/zh/docs/ops/ambient/install/helm-installation/index.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+1. [安装 Helm 客户端](https://helm.sh/docs/intro/install/)，版本要求 3.6 或更高。
+
+1. 配置 Helm 仓库：
+
+    {{< text syntax=bash snip_id=configure_helm >}}
+    $ helm repo add istio https://istio-release.storage.googleapis.com/charts
+    $ helm repo update
+    {{< /text >}}
+
+**有关 Helm 命令文档，请参阅 [Helm 仓库](https://helm.sh/docs/helm/helm_repo/)。**
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/install/helm-installation/index.md:28]:
```suggestion
*有关 Helm 命令文档，请参阅 [Helm 仓库](https://helm.sh/docs/helm/helm_repo/)。*
```

File: content/zh/docs/ops/ambient/install/helm-installation/index.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+并对稍后将安装的 ztunnel DaemonSet 之间的流量重定向进行配置。
+
+{{< text syntax=bash snip_id=install_cni >}}
+$ helm install istio-cni istio/cni -n istio-system \
+  -f @manifests/charts/istio-cni/ambient-values.yaml@
+{{< /text >}}
+
+### 安装发现组件 {#installing-the-discovery-component}
+
+`istiod` Chart 会安装 Istiod 的修订版。Istiod 是控制平面组件，用于管理和配置代理并在网格内进行流量路由。
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/install/helm-installation/index.md:53]:
```suggestion
`istiod` Chart 会安装 Istiod 的修订版。Istiod 是控制平面组件，用于管理和配置代理，以在网格内进行流量路由。
```

File: content/zh/docs/ops/ambient/install/helm-installation/index.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+    $ helm delete ztunnel -n istio-system
+    {{< /text >}}
+
+1. 删除 Istio 发现 Chart：
+
+    {{< text syntax=bash snip_id=delete_discovery >}}
+    $ helm delete istiod -n istio-system
+    {{< /text >}}
+
+1. 删除 Istio 基础 Chart：
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/install/helm-installation/index.md:136]:
```suggestion
1. 删除 Istio base Chart：
```

File: content/zh/docs/ops/ambient/install/helm-installation/index.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+    $ helm delete istio-cni -n istio-system
+    {{< /text >}}
+
+1. 删除 Istio ztunnel Chart：
+
+    {{< text syntax=bash snip_id=delete_ztunnel >}}
+    $ helm delete ztunnel -n istio-system
+    {{< /text >}}
+
+1. 删除 Istio 发现 Chart：
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/install/helm-installation/index.md:130]:
```suggestion
1. 删除 Istio discovery Chart：
```

File: content/zh/docs/ops/ambient/install/helm-installation/index.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+    {{< text syntax=bash snip_id=configure_helm >}}
+    $ helm repo add istio https://istio-release.storage.googleapis.com/charts
+    $ helm repo update
+    {{< /text >}}
+
+**有关 Helm 命令文档，请参阅 [Helm 仓库](https://helm.sh/docs/helm/helm_repo/)。**
+
+## 安装组件 {#installing-the-components}
+
+### 安装基础组件 {#installing-the-base-component}
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/install/helm-installation/index.md:32]:
```suggestion
### 安装 base 组件 {#installing-the-base-component}
```

File: content/zh/docs/ops/ambient/install/helm-installation/index.md [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+
+**CNI** Chart 会安装 Istio CNI 插件。它负责检测属于 Ambient Mesh 的 Pod，
+并对稍后将安装的 ztunnel DaemonSet 之间的流量重定向进行配置。
+
+{{< text syntax=bash snip_id=install_cni >}}
+$ helm install istio-cni istio/cni -n istio-system \
+  -f @manifests/charts/istio-cni/ambient-values.yaml@
+{{< /text >}}
+
+### 安装发现组件 {#installing-the-discovery-component}
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/install/helm-installation/index.md:51]:
```suggestion
### 安装 discovery 组件 {#installing-the-discovery-component}
```

**Review by windsonsea — APPROVED** [review]
lgtm

**Pull request merged**